openssl-dynamic/src/main/c/ssl.c

```diff
@@ -344,6 +344,11 @@ static long bio_java_bytebuffer_ctrl(BIO* bio, int cmd, long num, void* ptr) {
             return 1;
         case BIO_CTRL_FLUSH:
             return 1;
+        case BIO_C_SET_FD:
+#if defined(OPENSSL_IS_BORINGSSL) || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
good catch! Let me fix it

Reply:
Well actually no ... I think we always want to return 1 to make it a NOOP on openssl

Reply:
Yes, exactly what @normanmaurer said. Otherwise this will cause openssl to throw an error. It's not a problem for this to silently succeed in openssl's case.